python/pyarrow/tests/test_feather.py

```diff
@@ -36,12 +36,6 @@
     pass
 
 
-# TODO(wesm): The Feather tests currently are tangled with pandas
-# dependency. We should isolate the pandas-depending parts and mark those with
-# pytest.mark.pandas
-pytestmark = pytest.mark.pandas
```

Comment:
My "quick fix" was to just add the mark to all tests below (except the one I added). 
Not sure if there are better solutions for now. 
(I think many of the tests don't actually need pandas, if we would want to rewrite them slightly. Certainly now the pa.table(..) is more or less as friendly as pd.DataFrame(..) to create test data)

Reply:
Created a follow up: [ARROW-8355
[Python] Reduce the number of pandas dependent test cases in test_feather](https://issues.apache.org/jira/browse/ARROW-8355)